effect/extrinsic.test.ts

```diff
@@ -0,0 +1,150 @@
+import type { KeyringPair } from "https://deno.land/x/polkadot@0.0.8/keyring/types.ts";
+import { assertEquals, assertObjectMatch } from "../deps/std/testing/asserts.ts";
+import * as C from "../mod.ts";
+import * as t from "../test-util/mod.ts";
+import * as U from "../util/mod.ts";
+import { SendAndWatchExtrinsicProps } from "./mod.ts";
```

Comment:
You can delete this line and then access `SendAndWatchExtrinsicProps` from the namespace below:

```diff
async function collectExtrinsicEvents(
  { config, palletName, methodName, args }: Pick<
-   SendAndWatchExtrinsicProps,
+   C.SendAndWatchExtrinsicProps,
```